src/index.js

```diff
@@ -209,7 +209,7 @@ export default class OneSignal {
 
     /* N O T I F I C A T I O N S */
 
-    static postNotification(notificationObjectString, onSuccess=()=>{}, onFailure=()=>{}) {
+    static postNotification(notificationObjectString, onSuccess=function(){}, onFailure=function(){}) {
```

Comment:
Default Params are also in ES6, Chrome 49 so we need to not use this feature either.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

Reply:
Good catch